contrib/auto-render/README.md

```diff
@@ -55,7 +55,9 @@ function renderMathInElement(elem, options)
 `elem` is an HTML DOM element. The function will recursively search for text
 nodes inside this element and render the math in them.
 
-`options` is an optional object argument with the following keys:
+`options` is an optional object argument that can have the same keys as [the object 
+passed to `katex.render`](../../README.md##automatic-rendering-of-math-on-a-page),
```

Comment:
Relative links _should_ work on GitHub, but the absolute URL is clearer, so I'll switch to that.